fastlane/Fastfile

```diff
@@ -31,7 +31,7 @@ platform :ios do
 
   desc "Cocoapods library lint"
   lane :pod_lint do
-    pod_lib_lint(verbose: false, allow_warnings: true)
+    pod_lib_lint(verbose: false, allow_warnings: true, platforms: 'ios,osx,tvos')
```

Comment:
By default `pod lib lint` will build and run tests for all supported platforms. But had to exclude watchOS because of a [Cocoapods issue with Xcode 14](https://github.com/CocoaPods/CocoaPods/issues/11558), that is not fixed yet (even though the GH issue is closed).

This should be reverted once that issue is fixed.